src/data/padics/padic_numbers.lean

```diff
@@ -424,7 +425,7 @@ begin
   unfold norm at ⊢ hfgne, split_ifs at ⊢ hfgne,
   padic_index_simp [hfg, hf, hg] at ⊢ hfgne,
   apply padic_norm.add_eq_max_of_ne,
-  simpa [hf, hg, norm] using hfgne
+  simpa only using hfgne
```

Comment:
Can this just become `exact`? Or `convert`?

Reply:
thx, pushed